src/wmnds/patterns/cookies/cookies.js

```diff
@@ -0,0 +1,117 @@
+      return false;
+    }
+    return true;
+  };
+
+  const getCookiePolicy = () => JSON.parse(getCookie('cookies-policy'));
+
+  const updateCookiePreferences = () => {
+    if (document.querySelector('.wmnds-cookies-preferences')) {
+      hideCookieBanner();
+      const cookiesOptions = document
+        .querySelector('.wmnds-cookies-preferences')
+        .querySelectorAll('.wmnds-fe-checkboxes__input');
+      const currentOptions = [getCookiePolicy().essential, getCookiePolicy().functional, getCookiePolicy().performance];
+      for (let i = 0; i < cookiesOptions.length; i += 1) {
+        cookiesOptions[i].checked = currentOptions[i];
+      }
+    }
+  };
+
+  const setCookiePolicy = (essentialValue, functionalValue, performanceValue) => {
+    const cookieValue = {
+      essential: essentialValue,
+      functional: functionalValue,
+      performance: performanceValue
+    };
+    setCookie('cookies-policy', JSON.stringify(cookieValue), 181);
+    updateCookiePreferences();
+  };
+
+  const acceptAllCookies = () => {
+    setCookiePolicy(true, true, true);
+    setCookie('cookies-preference', true, 181);
+    hideCookieBanner();
+  };
+
+  const savePreferences = () => {
+    if (document.querySelector('.wmnds-cookies-manager__form')) {
+      const elements = document
+        .querySelector('.wmnds-cookies-manager__form')
+        .querySelectorAll('.wmnds-fe-checkboxes__input');
+      const selectedOptions = [];
+      for (let i = 0; i < elements.length; i += 1) {
+        selectedOptions[i] = elements.item(i).checked;
+      }
+      setCookiePolicy(...selectedOptions);
+      setCookie('cookies-preference', true, 181);
+    }
+  };
+
+  const cookiesScan = () => {
+    // if cookies-preference exist, hide cookie banner
+    if (checkCookie('cookies-preference')) {
+      hideCookieBanner();
+    }
+    // if not, create a default cookie, cookie banner will be show
+    else {
+      setCookiePolicy(true, false, false);
+    }
+
+    // verify if we are at Cookies Manager page and update the selected options to match the already created cookie
+    updateCookiePreferences();
+  };
+
+  // Creation of default Cookies permissions when the DOM is fully loaded
+  document.addEventListener('DOMContentLoaded', cookiesScan);
+
+  // When Accept all cookies button is triggered
+  const acceptAllCookiesBtn = document.querySelector('.wmnds-accept-all-cookies');
+  acceptAllCookiesBtn.addEventListener('click', acceptAllCookies);
+  acceptAllCookiesBtn.addEventListener('keydown', event => {
+    if (event.key === ' ' || event.key === 'Enter' || event.key === 'Spacebar') {
+      event.preventDefault();
+      acceptAllCookies();
+    }
+  });
+
+  // When Safe Preferences button is triggered
+  if (document.querySelector('.wmnds-cookies-manager__form')) {
+    const preferencesForm = document.querySelector('.wmnds-cookies-manager__form');
+    preferencesForm.addEventListener('submit', savePreferences);
+  }
```

Comment:
```suggestion
const cookieForm = document.querySelector('.wmnds-cookies-manager__form');
if (cookieForm)  cookieForm.addEventListener('submit', savePreferences);
```

Much simpler and don't forget to update the class name!